src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -422,4 +445,35 @@ function AppleHealthMenuItem(props: AppleHealthMenuItemProps) {
             </div>
         }
     </div>;
+}
+
+function WeatherMenuItem(props: { settings: DataCollectionSettings | null, participantInfo: ParticipantInfo | null, postalCodeSurveyName: string }){
+    let action = () => {
+        MyDataHelps.startSurvey(props.postalCodeSurveyName);
+    }
+
+    const postalCodeCustomFields = Object.keys(props.participantInfo ? props.participantInfo.customFields : {}).filter( (k) => k.endsWith('PostalCode'));
+    const postalCodes = [...postalCodeCustomFields.map((cf) => props.participantInfo?.customFields[cf]), props.participantInfo?.demographics.postalCode];
+    const postalCode = postalCodes.find((p) => !!p);
+
+    const indicator = postalCode ? 
+        <div className="mdhui-connect-devices-menu-connect">{postalCode}</div> :
```

Comment:
I'm kind of thinking we should `.join(', ')` all of the non-empty postal codes.  Being able to see both in the home/work scenario would be ideal.